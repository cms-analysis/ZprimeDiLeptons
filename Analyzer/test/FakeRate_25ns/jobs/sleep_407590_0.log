000 (407590.000.000) 09/13 23:20:30 Job submitted from host: <90.147.169.224:9618?addrs=90.147.169.224-9618&noUDP&sock=2441165_0900_3>
...
001 (407590.000.000) 09/13 23:36:41 Job executing on host: <90.147.168.154:60102>
...
006 (407590.000.000) 09/13 23:36:50 Image size of job updated: 120100
	77  -  MemoryUsage of job (MB)
	78576  -  ResidentSetSize of job (KB)
...
005 (407590.000.000) 09/13 23:36:54 Job terminated.
	(1) Normal termination (return value 1)
		Usr 0 00:00:08, Sys 0 00:00:01  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:08, Sys 0 00:00:01  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	98527  -  Run Bytes Sent By Job
	69209  -  Run Bytes Received By Job
	98527  -  Total Bytes Sent By Job
	69209  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :      214       75   8911106
	   Memory (MB)          :       77     2500      2500
...
