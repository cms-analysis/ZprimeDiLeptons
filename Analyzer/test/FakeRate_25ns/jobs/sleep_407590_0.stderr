mv: impossibile fare stat di `/home/condor/dir_11026/CMSSW803-Analyse_ZprimeToMuMu_13TeV.root': No such file or directory
mv: impossibile fare stat di `/home/condor/dir_11026/CMSSW803-Analyse_ZprimeToMuMu_13TeV_cand.txt': No such file or directory
